compose.yaml

```diff
@@ -1,11 +1,32 @@
 services:
-  ftsui:
+
+  # tak-server:
+  #     image: freetakserver:v2.1
+  #     ports:
+  #       - 8080:8080
+  #       - 8087:8087
+  #       - 8089:8089
+  #       - 8443:8443
+  #       - 9000:9000
+  #       - 19023:19023
+
+  tak-server-ui:
```

Comment:
please change this to show that this is Free TAK Server and not just the standard TAK server to prevent confusion with the government version